config/set/downgrade-php80.php

```diff
@@ -70,6 +71,7 @@
         DowngradePropertyPromotionRector::class,
         DowngradeNonCapturingCatchesRector::class,
         DowngradeStrContainsRector::class,
+        DowngradeMbStrContainsRector::class,
```

Comment:
unregister this, there is no `"mb_str_contains"`, only `str_contains`, that means that utilize `strpos` can be on purpose.

let user manually use if needed, dealing with user that may don't have mbstring extension or use strpos on purpose :)

user that use `mb_` functions know why they are using the `mb_` functions :)